tests/operator/helper.go

```diff
@@ -20,7 +20,12 @@ Package operator provides CNFCERT tests used to validate operator CNF facets.
 
 package operator
 
-import "strings"
+import (
+	"strings"
+
+	v1 "github.com/operator-framework/api/pkg/operators/v1"
```

Comment:
```suggestion
	operatorsv1 "github.com/operator-framework/api/pkg/operators/v1"
```

Something to make `v1` more explicit as an import.